docs/sles-installation-guide.md

```diff
@@ -0,0 +1,72 @@
+# Installing Intel® Clear Containers 3.0 on SLES* version 12 SP1
+
+This document _only_ covers installing on a
+[SLES](https://www.suse.com/products/server/) 12 SP1 system. Although,
+this guide could be used with more recent Service Packs, it has not been
+tested.
+
+Note:
+
+If you are installing on a system that already has Clear Containers 2.x
+installed, first read [the upgrading document](upgrading.md).
+
+## Required Setup
+
+The installation requires the current user to run the `sudo` command
+without specifying a password. Verify this with the following commands:
+
+```
+$ su -
+# echo "$some_user ALL=(ALL:ALL) NOPASSWD: ALL" | (EDITOR="tee -a" visudo)
+$ exit
+```
+
+## Installation steps
+
+1. Ensure the system packages are up-to-date:
+
+```
+$ sudo -E zypper update
+```
+2. Install Git:
+```
+$ sudo -E zypper -n install git
+
+```
+3. Create the installation directory and clone the repository:
+```
+$ mkdir -p $HOME/go/src/github/clearcontainers
+$ cd $HOME/go/src/github/clearcontainers
+$ git clone https://github.com/clearcontainers/runtime
+$ cd runtime
+```
+4. Run the installation script:
+
+```
+$ script -efc ./installation/sles-setup.sh
+```
+
+Note:
+
+- Running the installation script can take a long time as it needs to
+  download source packages and compile them.
+
+- Although it is not strictly necessary to run the installation
```

Comment:
Lines 54-56 suggested rewrite: "It is not strictly necessary to run the installation script using the `script(1)` command, but using this command writes a log of the installation to the file `typescript`."